docs/docs-content/clusters/cluster-management/backup-restore/enable-etcd-backup.md

```diff
@@ -0,0 +1,119 @@
+   | `cluster.config.etcd-snapshot-schedule-cron` | A cron expression that controls the time and frequency of scheduled backups. For example, the default value `0 */1 * * *` means the scheduled backup runs every hour. |
+   | `cluster.config.etcd-snapshot-retention`     | The maximum number of etcd backups to retain.                                                                                                                         |
+   | `cluster.config.etcd-disable-snapshots`      | Controls whether or not to disable scheduled etcd snapshots.                                                                                                          |
+   | `cluster.config.etcd-snapshot-dir`           | Specifies the directory where the etcd snapshots are saved. By default, this value is `/var/lib/rke2/rancher/server/db/snapshots`.                                    |
+
+   For example,
+
+   ```yaml
+   cluster:
+     config:
+       etcd-snapshot-schedule-cron: 0 */1 * * *
+       etcd-snapshot-retention: 12
+       etcd-disable-snapshots: false
+       etcd-snapshot-dir: /var/lib/rancher/rke2/server/db/snapshots
+   ```
+
+    </TabItem>
+
+    <TabItem value="k3s" label="K3S">
+
+   Use the following parameters to configure scheduled backups.
+
+   | Parameter                                    | Description                                                                                                                                                           |
+   | -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | `cluster.config.etcd-snapshot-schedule-cron` | A cron expression that controls the time and frequency of scheduled backups. For example, the default value `0 */1 * * *` means the scheduled backup runs every hour. |
+   | `cluster.config.etcd-snapshot-retention`     | The maximum number of etcd backups to retain.                                                                                                                         |
+   | `cluster.config.etcd-disable-snapshots`      | Controls whether or not to disable scheduled etcd snapshots.                                                                                                          |
+   | `cluster.config.etcd-snapshot-dir`           | Specifies the directory where the etcd snapshots are saved. By default, this value is `/var/lib/k3s/rancher/server/db/snapshots`.                                     |
+
+   For example,
+
+   ```yaml
+   cluster:
+     config:
+       etcd-snapshot-schedule-cron: 0 */1 * * *
+       etcd-snapshot-retention: 12
+       etcd-disable-snapshots: false
+       etcd-snapshot-dir: /var/lib/rancher/k3s/server/db/snapshots
+   ```
+
+    </TabItem>
+
+    </Tabs>
+
```

Comment:
I think we need to add a step that calls out, either update your active cluster profile and link to the cluster update page. Or, if you have not deployed a cluster yet, go ahead and deploy a cluster. That way we provide guidance on the next step to the user.